src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/events/AppointmentEventPublisher.kt

```diff
@@ -22,9 +22,10 @@ class AppointmentEventPublisher(
   private val applicationEventPublisher: ApplicationEventPublisher,
   private val locationMapper: LocationMapper
 ) {
-
-  fun appointmentNotAttendedEvent(appointment: ActionPlanAppointment) {
-    applicationEventPublisher.publishEvent(AppointmentEvent(this, AppointmentEventType.NO_ATTENDANCE, appointment, getAppointmentAttendanceURL(appointment)))
+  fun attendanceRecordedEvent(appointment: ActionPlanAppointment) {
+    applicationEventPublisher.publishEvent(
+      AppointmentEvent(this, AppointmentEventType.ATTENDANCE_RECORDED, appointment, getAppointmentAttendanceURL(appointment))
```

Comment:
`AppointmentAttendanceEvent`?

Reply:
the naming here with the type argument fits in with the existing patterns used for other event publishers. i'd rather keep it consistent.

Reply:
👍 makes sense, thanks